packages-ts/gauntlet-cosmos-contracts/package.json

```diff
@@ -21,7 +21,10 @@
     "preinstall": "node ../../scripts/require-yarn.js",
     "gauntlet": "ts-node ./src/index.ts",
     "lint": "tsc",
-    "test": "SKIP_PROMPTS=true jest --runInBand",
+    "dev:test:wasmd-up": "yarn ts-node ./test/scripts/startWasmdAndUpload.ts",
```

Comment:
my dev workflow is to run `dev:test:wasmd-up` up, then just run `dev:test` so I don't need to start wasmd